planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/OptimizerInitialDeployment.java

```diff
@@ -262,7 +262,10 @@ private void addWorkloadAndPoolSizeBoundsForScalableModules(Solution solution,
             log.debug("Adding upperWkl= " + upperWklBound + " lowerWkl=" + lowerWklBound + " poolsize=" + maxPoolSize);
             YAMLoptimizerParser.addScalingPolicyToModule(entry.getKey(), baseAppMap, lowerWklBound, upperWklBound, 1,
                   maxPoolSize);
-            YAMLoptimizerParser.changeModuleToScalableType(entry.getKey(), baseAppMap);
```

Comment:
If this method is removed, the types of modules remain unaltered from the declared in the AAM. 

"Scalable type" was for example the DynamicWebAppCluster. 
Some module (of any type, for example database.mysql.MySqlNode) are marked in the AAM as scalable, and the output of the optimizer can give the type of resources to use and the conditions when the module should scale-in/out. 
Once the previous conditions are computed, the type of the modules which got these scaling conditions was changed on purpose to a scalable type (this is, to the ControlledDynamicWebAppCluster). 

This behavior was not a bug or accidental, but it was done on purpose (it was not a bug, it was a feature). Actually, it had its own tests that checked that the type had been correctly changed. During the DAM learning process, in some moment we had thought that this was the suitable solution to implement. 

Since the change of the types declared in the AAM was done in method changeModuleToScalableType, just  removing the line of the invocation to this method solves the issue of the changes of types. 

Then, additionally, I removed the tests that checked that the types of modules had been correctly changed because now they are not changed. 

Does it answer your questions?